src/flownet/parameters/_equilibration.py

```diff
@@ -125,10 +125,25 @@ def render_output(self) -> Dict:
             )
 
             if self._rsvd is not None:
+                df_rsvd = pd.read_csv(self._rsvd)
+                if len(df_rsvd.columns) == 3:
+                    df_rsvd.columns = ["depth", "rs", "eqlnum"]
```

Comment:
Here we assume that the columns are in this order. Would it be safer to let the user _name_ the columns. We then read the column names (case insensitive) and use those. If the column names are spelled wrongly we return an error saying: "Didn't find this or that".

That's maybe somewhat less error prone?

Reply:
Added checking of the csv file in _config_parser.py, so when we arrive at this part of the code, the file should be known to be ok